src/components/ViewDeck/ViewDeck.js

```diff
@@ -97,228 +80,205 @@ class ViewDeck extends Component {
-      JSON.parse(fCard.api_data)
-    );
-    console.log("parsedFeaturedCard", parsedFeaturedCard);
-    const featuredUri = parsedFeaturedCard.map(
-      (fUri) => fUri.image_uris.normal
-    );
-
-    this.countQty(includedCards);
-
-    const parsedCards = this.parseCards(includedCards);
-    console.log(
-      "🚀 ~ file: ViewDeck.js ~ line 168 ~ ViewDeck ~ render ~ parsedCards",
-      parsedCards
-    );
-
-    const devotion = this.devotion(parsedCards);
-    console.log(
-      "🚀 ~ file: ViewDeck.js ~ line 204 ~ ViewDeck ~ render ~ devotion",
-      devotion
-    );
-
-    const cmc = this.cmc(parsedCards);
-    console.log(
-      "🚀 ~ file: ViewDeck.js ~ line 215 ~ ViewDeck ~ render ~ cmc sans land",
-      cmc
-    );
-
-    return (
-      <div>
-        <h1 className="editDeckHeader">
-          Viewing {this.props.reduxStore.selectedDeck.deckname} from{" "}
-          {this.props.reduxStore.user.username}{" "}
-        </h1>
-        {/* <h1 id="editDeckHeader">Total Cards: {this.state.deckQty}</h1> */}
-
-        <div className="viewDeckEditDeckView">
-          <div>
-            <img
-              className="featuredViewDeckCard"
-              src={this.state.hoverCard}
-              width="200px"
-              height="280"
-            />
-          </div>
-
-          <br />
-          <h4 className="editDeckHeader">Total Cards: {this.state.deckQty}</h4>
+  const includedCards = props.reduxStore.cardList.filter(
+    (card) => card.deckid === props.reduxStore.selectedDeck.id
+  );
+  // console.log("includedCards:", includedCards);
+  const featuredCard = includedCards.filter(
+    (featured) => featured.is_featured === true
+  );
+  const parsedFeaturedCard = featuredCard.map((fCard) =>
+    JSON.parse(fCard.api_data)
+  );
+  // console.log("parsedFeaturedCard", parsedFeaturedCard);
+  const featuredUri = parsedFeaturedCard.map((fUri) => fUri.image_uris.normal);
+
+  countQty(includedCards);
+
+  const parsedCards = parseCards(includedCards);
+
+  const devotion = devotionCalc(parsedCards);
+
+  const cmc = cmcCalc(parsedCards);
+
+  return (
+    <div>
+      <h1 className="editDeckHeader">
+        Viewing {props.reduxStore.selectedDeck.deckname} from{" "}
+        {props.reduxStore.user.username}{" "}
+      </h1>
+
+      <div className="viewDeckEditDeckView">
+        <div>
+          <img
+            className="featuredViewDeckCard"
+            src={hoverCard}
+            width="200px"
+            height="280"
+          />
+        </div>
 
-          <table>
-            <thead>
-              <tr>
-                <th></th>
-                {/* Hovercard^ */}
-                <th>Quantity</th>
-                <th className="nameTh">Card Name</th>
-                {/* <th>isCMDR?</th> */}
-                <th></th>
-                {/* isFeatured? */}
-                <th></th>
-                {/* Type */}
-                {/* <th>DELETE</th> */}
+        <br />
```

Comment:
I would urge you to get away from `<br/>`. Use css to position your html and create space.